src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -64,13 +64,13 @@ public class SignInManager<TUser> where TUser : class
         {
             if (Options.SignIn.RequireConfirmedEmail && !(await UserManager.IsEmailConfirmedAsync(user)))
             {
-                return await LogResultAsync(false, user);
+                return await false.WithLoggingAsync(this, user);
             }
             if (Options.SignIn.RequireConfirmedPhoneNumber && !(await UserManager.IsPhoneNumberConfirmedAsync(user)))
             {
-                return await LogResultAsync(false, user);
+                return await false.WithLoggingAsync(this, user);
```

Comment:
I agree with that. We can a split where logging after Task<IdentityResult> can be an extension method while for bool results it can be in the Manager class itself.

Having an extension method makes it easy to have a fluent call that's all